packages/devtools_app/lib/src/screens/provider/instance_viewer/instance_providers.dart

```diff
@@ -210,7 +203,7 @@ Future<EnumInstance> _tryParseEnum(
   final name = await nameInstanceFuture;
   if (name.kind != InstanceKind.kString) return null;
 
-  final nameSplit = name.valueAsString.split('.');
+  final nameSplit = name.valueAsString!.split('.');
```

Comment:
How about adding null safety:`(name.valueAsString ?? '').split('.');`

Reply:
I don't like the idea. If `valueAsString` is null here, there's likely a problem. So `?? ''` would only be silencing the error imo

Reply:
Overall I don't like how `Instance` works. 

I'd prefer if we could do:

```dart
if (name is! StringInstance)  return null;

name.valueAsString; // non-nullable
```

This would avoid making all the possible properties of `Instance` nullable.